src/LocProject.json

```diff
@@ -0,0 +1,57 @@
+{
+  "Projects": [
+    {
+      "LanguageSet": "AzureKatal_Languages",
+      "LocItems": [
+        {
+          "SourceFile": "src\\Microsoft.SqlTools.ServiceLayer\\Localization\\sr.xlf",
+          "LclFile": "src\\Microsoft.SqlTools.ServiceLayer\\Localization\\LCL\\{Lang}\\sr.xlf.lcl",
+          "CopyOption": "LangIDOnName",
+          "OutputPath": "src\\Microsoft.SqlTools.ServiceLayer\\Localization"
+        },
+		{
```

Comment:
fix tab spacing

Reply:
This was deliberate to match ADS, but I'll revert it